test/client/fixtures/sandbox/shadow-ui-test.js

```diff
@@ -340,11 +340,22 @@ test('body.querySelector', function () {
     pageElem.className = 'test-class cli2';
     document.body.appendChild(pageElem);
 
-    uiElem   = document.body.querySelector('.cl1');
+    uiElem   = document.body.querySelector('.cli');
     pageElem = document.body.querySelector('.cli2');
 
     ok(!uiElem);
     strictEqual(pageElem.id, 'pageElem');
+
```

Comment:
Let's create a separate test for document and element's querySelector with th following scenario:
- create shadowUI and regular elements with equal classes.
- insert element so that shadow is first element and reqular is second
- querySelector elements by class and check that shadowUI element is skipping and found the second reqular element